### PR TITLE
[Inspiration screens]: Product Page & Twitter screens

### PR DESCRIPTION
**Pull request by maeschli**
## Description
Created Demo Screens in Inspiration Screens section

## Changelog
```
demo/src/index.js
demo/src/screens/MenuStructure.js
demo/src/screens/realExamples/ProductPage/index.tsx - new one
demo/src/screens/realExamples/Twitter/index.tsx - new one
demo/src/screens/realExamples/index.js
```

### CODE REVIEW

File: demo/src/index.js [hunk 1]
```diff
@@ -241,6 +241,9 @@ module.exports = {
       // wrapperScreens
       get TouchableOpacityScreen() {
         return require('./screens/componentScreens/TouchableOpacityScreen').default;
+      },
+      get ProductPage() {
+        return require('./screens/realExamples/ProductPage');
```
Comment (ethanshar) [inline demo/src/index.js:246]: Let's move this export to be part of `// realExamples` 
just above `// wrapperScreens`

File: demo/src/screens/realExamples/ProductPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          <ActionSheet
+            title={'Select a Color'}
+            options={[
+              {label: colorOptions.red.name, onPress: () => this.pickOption(colorOptions.red)},
+              {label: colorOptions.green.name, onPress: () => this.pickOption(colorOptions.green)},
+              {label: colorOptions.blue.name, onPress: () => this.pickOption(colorOptions.blue)},
+            ]}
+            visible={isColor}
+            onDismiss={()=> this.setState({isColor: false})}
+          />
```
Comment (ethanshar) [inline demo/src/screens/realExamples/ProductPage/index.tsx:96]: Why not use the Picker component?

File: demo/src/screens/realExamples/ProductPage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+          <ActionSheet
+            title={'Select a Size'}
+            options={[
+              {label: sizeOptions.s.name, onPress: () => this.pickOption(sizeOptions.s)},
+              {label: sizeOptions.m.name, onPress: () => this.pickOption(sizeOptions.m)},
+              {label: sizeOptions.l.name, onPress: () => this.pickOption(sizeOptions.l)},
+            ]}
+            visible={isSize}
+            onDismiss={()=> this.setState({isSize: false})}
+          />
```
Comment (ethanshar) [inline demo/src/screens/realExamples/ProductPage/index.tsx:116]: Same here, consider using the Picker component

File: demo/src/screens/realExamples/ProductPage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+      
+          <Button style={{marginVertical: 40}} label={'Add to Cart'} />
```
Comment (ethanshar) [inline demo/src/screens/realExamples/ProductPage/index.tsx:118]:
```suggestion
          <Button marginV-40 label={'Add to Cart'} />
```

File: demo/src/screens/realExamples/ProductPage/index.tsx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+              />
+            ))
+          }
+        </ScrollView>
+      </ScrollView>
+    );
+  }
+}
+
+export default Product;
```
Comment (ethanshar) [inline demo/src/screens/realExamples/ProductPage/index.tsx:138]: Missing empty space at the end
```suggestion
export default Product;

```

**Review by ethanshar — COMMENTED** [review]
Looks great! 
Wrote a few comments

File: demo/src/screens/realExamples/ProductPage/index.tsx (hunk 2 above)
Comment (maeschli) [reply demo/src/screens/realExamples/ProductPage/index.tsx:96]: I thought that Picker doesn't have ActionSheet, but seems like it has in custom options

Comment (ethanshar) [reply demo/src/screens/realExamples/ProductPage/index.tsx:96]: Yes, technically you achieve the same behavior with custom item render.

**Comment by ethanshar** [comment]
@maeschli Let me know if you still have time to fix my comments. 
I can always merge and we can handle it later on in a different PR

**Review by ethanshar — APPROVED** [review]

**Pull request merged**